### PR TITLE
Comment Fixes

### PR DESCRIPTION
**Pull request by yoveian**

### CODE REVIEW

File: cfg.go [hunk 1]
```diff
@@ -1,10 +1,41 @@
...
+const (
+	SingleTxMode = "single"
+	BatchTxMode  = "batch"
+)
 
 type Config struct {
 	Network  string
-	Contract string
 	TxMode   string
 	ChainID  string
```
Comment (rengen12) [inline cfg.go]: get chain id from const. Testnet `atlantic-2`, mainnet `pacific-1`

File: client.go [hunk 2]
```diff
@@ -93,28 +97,95 @@ func NewClient(cfg Config) (c *Client, err error) {
...
 
+	tmClient, err := client.NewClientFromNode(cfg.RPCHost)
+	if err != nil {
+		return nil, fmt.Errorf("NewClientFromNode error: %w", err)
+	}
+
 	conn, err := grpc.NewClient(cfg.GRPCHost, grpc.WithTransportCredentials(credentials.NewClientTLSFromCert(nil, "")))
 	if err != nil {
 		return nil, fmt.Errorf("grpc.Dial: %s %s", cfg.GRPCHost, err)
 	}
```
Comment (rengen12) [inline client.go]: create connection networks last

File: client.go [hunk 3]
```diff
@@ -93,28 +97,95 @@ func NewClient(cfg Config) (c *Client, err error) {
...
+		WithKeybase(clientCtx.Keyring).
+		WithTxConfig(clientCtx.TxConfig).
+		WithAccountRetriever(clientCtx.AccountRetriever).
+		WithSimulateAndExecute(true).
+		WithGasAdjustment(1.1).
+		WithChainID(clientCtx.ChainID).
+		WithSignMode(signing.SignMode_SIGN_MODE_DIRECT).
+		WithGasPrices(DefaultGasPriceWithDenom)
+
+	sgn := &sign{
```
Comment (rengen12) [inline client.go:135]: should it be a separate structure?

Comment (yoveian) [reply client.go:135]: I think if we will keep it separate - it would be easier to read

File: client.go (hunk 2 above)
Comment (yoveian) [reply client.go]: done

File: cfg.go (hunk 1 above)
Comment (yoveian) [reply cfg.go]: done

File: client_instantiate.go [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+func (c *Client) Instantiate(codeID uint64, label, instantiateMsg string, funds []sdktypes.Coin) (string, error) {
+	if instantiateMsg == "" {
+		return "", errors.New("message code is empty")
+	}
+
+	if label == "" {
+		return "", errors.New("label is empty")
+	}
+
+	if len(funds) == 0 {
```
Comment (rengen12) [inline client_instantiate.go]: funds can be empty. Let's give this opportunity

Comment (yoveian) [reply client_instantiate.go]: done

File: client_broadcast.go [hunk 5]
```diff
@@ -74,18 +23,24 @@ func (c *Client) asyncBroadcastMsg(msgs ...sdk.Msg) (*txtypes.BroadcastTxRespons
...
+		for i := range 5 {
 			if err == nil {
 				break
 			}
 
 			if strings.Contains(err.Error(), "account sequence mismatch") {
-				c.syncNonce()
+				err = c.syncNonce()
+				if err != nil {
+					return nil, fmt.Errorf("syncNonce: %s", err)
```
Comment (rengen12) [inline client_broadcast.go]: hmm, you might get 429 error or service temporary unavailable. Maybe we'll try again?? Just `continue`

Comment (yoveian) [reply client_broadcast.go]: continue with log.Warnf() ?

Comment (rengen12) [reply client_broadcast.go]: yep

Comment (yoveian) [reply client_broadcast.go]: done

**Pull request merged**